src/kind.ts

```diff
@@ -0,0 +1,74 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const kindJobImage = "radumatei/golang-kind:1.11-0.4";
+
+export class KindJob extends Job {
+    constructor(name: string, image?: string) {
+        if (image == undefined) {
+            image = kindJobImage;
+        }
+        super(name, image);
+
+        // kind needs to run as a privileged pod
+        this.privileged = true;
+
+        // kind needs to add the following volumeMounts to function properly
+        // the Brigade project must enable `allowMountHosts`
+        this.volumes = [
+            {
+                name: "modules",
+                hostPath: {
+                    path: "/lib/modules",
+                    type: "Directory"
+                }
+            },
+            {
+                name: "cgroup",
+                hostPath: {
+                    path: "/sys/fs/cgroup",
+                    type: "Directory"
+                }
+            },
+            {
+                name: "docker-graph-storage",
+                emptyDir: {}
+            }
+        ];
+        this.volumeMounts = [
+            {
+                name: "modules",
+                mountPath: "/lib/modules",
+                readOnly: true
+            },
+            {
+                name: "cgroup",
+                mountPath: "/sys/fs/cgroup"
+            },
+            {
+                name: "docker-graph-storage",
+                mountPath: "/var/lib/docker"
+            }
+        ];
+
+        // to add your own tasks to this job, use job.tasks.push()
+        this.tasks = [
```

Comment:
The last task should always be `kind delete cluster`, and it should be executed even in a task failed.

Reply:
What is the behavior if `kind delete cluster` isn't invoked?  Lingering resources on the k8s nodes?

I wonder if this prompts a potential new feature in brigade - something like the concept of `defer` or `trap` - for tasks that need to be run regardless?  (Or maybe we can already do so?)